.github/workflows/dev.yml

```diff
@@ -42,8 +42,7 @@ jobs:
           node-version: "14"
       - name: Prettier check
         run: |
-          # if you encounter error, try rerun the command below with --write instead of --check
-          # and commit the changes
+          # if you encounter error, rerun the command below and commit the changes
```

Comment:
driveby cleanup (the command below actually uses `--write` rather than `--check` already